src/screens/ComposeScreen.js

```diff
@@ -256,8 +259,15 @@ class ComposeScreen extends Component {
               route={'AddGroup'} />
           </View>
           
-          <GreyLine boxStyle={[styles.lineSolid, { marginBottom: 20 }]} />
+          <GrayLine boxStyle={[styles.lineSolid, { marginBottom: 20 }]} />
         </View>
+       }
+        <KeyboardAccessoryNavigation
+          bumperHeight={300}
```

Comment:
Что это за число? Почему именно 300?

Reply:
Это высота серого фона на случай, если анимация клавиатуры срабатывает быстрее, чем плашка accessory успевает скрыться. Иначе белую подложку видно. Может, конечно, опять же, только в девелопменте, но вреда от того, чтобы задать это значение нет. Из доки: Bumper height to prevent visual glitches if animation couldn't keep up with the keyboard animation.

Reply:
Просто любопытно насколько это гибкое значение. Именно 300.

Reply:
Проверила на телефоне, с клавиатурой и кнопкой все ок, а вот контент весь теперь не помещается.

Reply:
300 это где-то высота клавиатуры

Reply:
Так вред в итоге есть? Что значит контент теперь весь не помещается? :)
Касательно числа. Я думаю ты понимаешь что это magic number. 300 как ты сама говоришь "где-то" высота клавиатуры. Где-то я так понимаю это здесь и сейчас. Не факт что она такая будет когда выйдет обновление iOS, не такая она будет на андроиде, если мы захотим его поддерживать тоже(а это скорее всего произойдет), не такая она будет если это кастомная клавиатура(не знаю можно ли так на iOS).
Я лично не использовал такой аттрибут, и нареканий вроде как ни у кого не было. Но я понял для чего в целом ты этим воспользовалась.

Reply:
Кстати чтоб проверить скорость работы, как я уже писал в чате, можно переключатель выставить в production, и запустить потом приложение

Reply:
Я новую разметку не тестировала на старом айфоне, сейчас залила с изменениями. Сейчас проверяю скорость работы.

Reply:
Убрала аттрибут bumberHeight